frontend/src/styles/elements/elements.scss

```diff
@@ -12,7 +12,7 @@ p {
 h1 {
   font-size: $default-h1;
   font-weight: $bold;
-  color: $font-black;
+  color: $h1-color;
```

Comment:
When the typography gets changed, they will likely be custom classes instead of h tags, so the naming `h1-color` may no longer work at that point, but that can be addressed when that refactor occurs.